keyboards/handwired/dactyl_manuform/5x7_2_6/keyboard.json

```diff
@@ -0,0 +1,45 @@
+{
+    "manufacturer": "Nikolai Grigoriev",
+    "keyboard_name": "Dactyl Manuform 5x7 2-key last row 6-key thumb cluster",
+    "maintainer": "ngrigoriev",
+    "bootloader": "caterina",
```

Comment:
```suggestion
```

Reply:
Will do, but, unless I am missing something, https://docs.qmk.fm/pr_checklist literally says:

info.json

    With the move to ...
    the mandatory elements for a minimally complete info.json at present are:
...
        Microcontroller and bootloader